packages/odyssey-storybook/src/components/Box/Box.stories.tsx

```diff
@@ -20,11 +20,17 @@ export default {
   component: Source,
 };
 
-const Template: Story<BoxProps> = (args) => (
-  <Box {...args} tabIndex={0}>
-    Box
-  </Box>
-);
+const Template: Story<BoxProps> = (args) => {
+  const {
+    as, // eslint-disable-line @typescript-eslint/no-unused-vars
```

Comment:
hmmm, don't we have stories for Box that make use of the the polymorphic `as` API? It feels incorrect to avoid using it here?